cmd/controller/main.go

```diff
@@ -57,8 +71,7 @@ func main() {
 	cfg.QPS = 2 * cfg.QPS
 	cfg.Burst = 2 * cfg.Burst
 
-	flag.Parse()
-	ctx := injection.WithNamespaceScope(signals.NewContext(), *namespace)
+	ctx := injection.WithNamespaceScope(signals.NewContext(), "")
```

Comment:
We can omit this (if the context value isn't set, this returns "" anyway).

https://github.com/knative/pkg/blob/7ecd5485cbc68c6cac94f626d3c7b11ff3b1275a/injection/context.go#L45-L51